### DISCUSSION TITLE
Can it be used in termux?

### DISCUSSION THREAD

**Discussion by drakewebs2017** [masterking32/MasterHamsterKombatBot#73] [Q&A] [answered]
Hello, I would like to know if this repository can be used in Termux so I can use it on my mobile. I await your response. Thank you.

**Comment by Microzibert** [comment, marked as answer]
Yes you can
```python
pkg install python
``` 
Then copy the path to the program folder
```
cd /Your/Path/to/folder 
``` 
check if you are in the right folder with the command _**ls**_ this will show a list of files in the folder

```
pip install requests
``` 
```
pip install colorlog
``` 
Now you can start the program 
```python
python main.py
```

**Reply by drakewebs2017** [reply]
thank you!! ready, I already did it, greetings
